check_stability.py

```diff
@@ -582,6 +622,11 @@ def get_parser():
                         # This is a workaround to get what should be the same value
                         default=os.environ.get("TRAVIS_REPO_SLUG").split('/')[0],
                         help="Travis user name")
+    parser.add_argument("--output-cap",
```

Comment:
Sure (though I'll point out that this increases the noun/verb ambiguity in the word "output")